gprofiler/profilers/java.py

```diff
@@ -902,6 +902,7 @@ def _check_async_profiler_loaded(self, process: Process) -> bool:
         return False
 
     def _profile_process(self, process: Process, duration: int, spawned: bool) -> ProfileData:
+        container_name = self._profiler_state.get_container_name(process.pid)
```

Comment:
Can move next to app_metadata and appid as suggested https://github.com/Granulate/gprofiler/pull/667/files#r1116345068